.appveyor.yml

```diff
@@ -1,28 +1,66 @@
-version: "{build}"
+version: 1.36.0.{build}
 os: Visual Studio 2017
 clone_folder: C:\Projects\MaterialX
 
 environment:
   matrix:
     - CMAKE_PLATFORM: "Visual Studio 14 2015"
       PYTHON: C:\Python26
+      VS_PLATFORM: x86
+      VS_COMPILER: vs2015
     - CMAKE_PLATFORM: "Visual Studio 14 2015 Win64"
       PYTHON: C:\Python27-x64
+      VS_PLATFORM: x64
+      VS_COMPILER: vs2015
     - CMAKE_PLATFORM: "Visual Studio 15 2017"
       PYTHON: C:\Python35
+      VS_PLATFORM: x86
+      VS_COMPILER: vs2017
     - CMAKE_PLATFORM: "Visual Studio 15 2017 Win64"
       PYTHON: C:\Python36-x64
+      VS_PLATFORM: x64
+      VS_COMPILER: vs2017
 
 configuration:
   - Release
-    
+
 install:
   - set PATH=%PYTHON%;%PATH%
-  
+
 build_script:
   - mkdir build
   - cd build
   - cmake -DMATERIALX_BUILD_PYTHON=ON -DMATERIALX_WARNINGS_AS_ERRORS=ON -G "%CMAKE_PLATFORM%" ..
   - cmake --build . --target install --config %CONFIGURATION%
   - ctest --output-on-failure --build-config %CONFIGURATION%
   - cmake -E chdir ../python/MaterialXTest python main.py
+
+after_build:
+    # Go to install directory and zip it up.  Currently marked as pre-release
+    - cd C:\Projects\MaterialX\build\installed
+    - 7z a -r C:\Projects\MaterialX\MaterialX-Windows-%APPVEYOR_REPO_BRANCH%-%VS_PLATFORM%-%VS_COMPILER%-%APPVEYOR_BUILD_NUMBER%.zip .
```

Comment:
Using absolute path here, is that intentional?

Reply:
Yes. It's the folder we set up top to clone into + the explicitly created "build" subfolder. "installed" is the default install folder which I didn't bother to override.